places/berlin.ini

```diff
@@ -111,3 +111,28 @@ url_path = div/div/div[2]/h2/a/@href
 title_path = div/div/div[2]/h2/a
 location_path = div/div/div[2]/address:(?<=,)[^,]*
 rooms_path = div/div/div[2]/ul/li[1]/span[2]
+
+[company:www.blp-management.com]
+url = https://blp-management.com/immobilien/?post_type=immomakler_object&vermarktungsart=miete&nutzungsart=wohnen&typ=wohnung&ort=berlin&center&objekt-id&collapse=in
+ad_path = .//div[@class='property-container']
+url_path = div[2]/h3/a/@href
+title_path = div[2]/h3/a
+location_path = div[2]/div
```

Comment:
The location contains the word `Etagenwohnung`, e.g. `10709 Berlin, Etagenwohnung`. Could you please add a regular expression pattern to remove it? :) We're doing something similar for Kurtztke Immobilien. (see also the docs in `flatdir/res/default.ini`)